bin/node/cli/tests/common.rs

```diff
@@ -32,3 +39,25 @@ pub fn wait_for(child: &mut Child, secs: usize) -> Option<ExitStatus> {
 
 	None
 }
+
+pub fn run_command_for_a_while(base_path: &Path, dev: bool) {
```

Comment:
Missing docs and "a while" is a very vague description :D

Reply:
I added some doc with more precise description